ci/travis_script_manylinux.sh

```diff
@@ -39,4 +44,21 @@ conda activate $CONDA_ENV_DIR
 
 pip install -q tensorflow
 pip install "dist/`ls dist/ | grep cp36`"
-python -c "import pyarrow; import tensorflow"
+popd
+
+# # Test optional dependencies and the presence of tensorflow
+python -c "
+import sys
+import pyarrow
+import pyarrow.orc
+import pyarrow.parquet
+import pyarrow.plasma
+import tensorflow
```

Comment:
Aren't we crashy when Tensorflow is loaded in the same process?

Reply:
I guess this is supposed to test that.

Reply:
We should not crash, there is some special safeaguarding code for that.

Reply:
But it doesn't always work, see https://issues.apache.org/jira/browse/ARROW-4272

Reply:
So shouldn't we remove this import?

Reply:
Why should We? If We want to test that both arrow and tensorflow can live in the same interperter.

Reply:
As you prefer... but it will probably fail at one point or the other, because Tensorflow publishes rogue wheels.